### DISCUSSION TITLE
vim.info is nil?

### DISCUSSION THREAD

**Discussion by alex-popov-tech** [mrjones2014/smart-splits.nvim#109] [Q&A] [answered]
hey! im trying to call resize mod method, and getting `vim.info is nil error`, so i went to neovim repo, and i could not find this function...

could you please elaborate on why its there, and/or when/how could it be nil?
<img width="973" alt="CleanShot 2023-05-03 at 01 58 40@2x" src="https://user-images.githubusercontent.com/21224705/235803118-976d37d2-75db-43ba-a344-5f6129fd953e.png">

**Comment by mrjones2014** [comment, marked as answer]
Whoops, looks like that was a typo. I think most people use `silent = true` which is probably why this didn't come up sooner.

Anyway I've pushed a fix to master. If you pin to a semantic version, it'll be available whenever I tag the next release. For now, you can use master.

**Reply by alex-popov-tech** [reply]
thanks, working now
